open-machine-learning-jupyter-book/prerequisites/python-programming-advanced.md

```diff
@@ -451,69 +475,84 @@ Changing global or nonlocal variables from within an inner function might be a B
 ```{code-cell}
 # pylint: disable=invalid-name
 test_variable = 'initial global value'
+```
 
+Scopes and Namespaces Example
 
-def test_function_scopes():

```

Comment:
You should only move this function itself's comment to  be as text. Leave what is inside  the func as